zonal: flake8 F401, E501 + isort drift

## Summary

The style sweep against `xrspatial/zonal.py` (2705 LOC, 57 commits) reports
drift since the last cleanup pass.

## flake8

```
xrspatial/zonal.py:39:1: F401 'xrspatial.utils.not_implemented_func' imported but unused
xrspatial/zonal.py:455:101: E501 line too long (117 > 100 characters)
```

`not_implemented_func` is only present on the import line; there are no
call sites, so removing it is safe.

The E501 line is a one-liner that builds a `dd.concat` of zone stats — it
wraps cleanly across two lines.

## isort

`isort --check-only --diff` reports the `xrspatial.utils` import block needs
consolidation: the trailing `has_dask_array` import gets merged back in, the
names are alphabetised, and the blank line above the `# 3rd-party` comment
gets trimmed.

## Cat 5 grep

Clean — no bare `except:`, no mutable default args, no shadowed builtins.

## Scope

Three findings, one PR:
- Cat 1 (E501): wrap the long line.
- Cat 3 (F401): drop the unused name.
- Cat 4 (isort): consolidate the `xrspatial.utils` import block.

No behavioural change.

--- a/.claude/sweep-style-state.csv
+++ b/.claude/sweep-style-state.csv
@@ -1,3 +1,4 @@
 module,last_inspected,issue,severity_max,categories_found,notes
 geotiff,2026-05-27,2481,HIGH,1;3;4,"Bundled 387 flake8 + ~30 isort fixes since #2285/#2430. F401 x9, F811 x6, F841 x3. E501 x250 (mostly wrapped, 3 file-scope imports keep noqa: E402+E501). E252 x62, blank-line cluster, E128/E127 indents. importorskip imports use # noqa: E402. Cat 5 grep clean."
 rasterize,2026-05-27,2503,HIGH,1;3,F401 line 15 + F811 line 1193 (paired: local import warnings shadowed unused module-level import); E306 line 1775 (nested @cuda.jit). isort clean. Cat 5 grep clean. Fix in PR #2507.
+zonal,2026-05-27,2522,HIGH,1;3;4,"F401 not_implemented_func (line 42, only present on import line). E501 line 455 (dd.concat one-liner, 117 chars) wrapped across 3 lines. isort: consolidated xrspatial.utils block (merged has_dask_array, dropped not_implemented_func, alphabetised, trimmed extra blank line). Cat 5 grep clean. 125 zonal tests pass."

--- a/xrspatial/zonal.py
+++ b/xrspatial/zonal.py
@@ -5,7 +5,6 @@ import copy
 from math import sqrt
 from typing import Callable, Dict, List, Optional, Union
 
-
 # 3rd-party
 try:
     import dask
@@ -36,12 +35,9 @@ except ImportError:
         ndarray = False
 
 # local modules
-from xrspatial.utils import (
-    ArrayTypeFunctionMapping, _validate_raster, cuda_args, has_cuda_and_cupy,
-    is_cupy_array, is_dask_cupy,
-    ngjit, not_implemented_func, validate_arrays,
-)
-from xrspatial.utils import has_dask_array
+from xrspatial.utils import (ArrayTypeFunctionMapping, _validate_raster, cuda_args,
+                             has_cuda_and_cupy, has_dask_array, is_cupy_array, is_dask_cupy, ngjit,
+                             validate_arrays)
 
 TOTAL_COUNT = '_total_count'
 
@@ -452,7 +448,10 @@ def _stats_dask_numpy(
             )
 
     # generate dask dataframe
-    stats_df = dd.concat([dd.from_dask_array(s) for s in stats_dict.values()], axis=1, ignore_unknown_divisions=True)
+    stats_df = dd.concat(
+        [dd.from_dask_array(s) for s in stats_dict.values()],
+        axis=1, ignore_unknown_divisions=True,
+    )
     # name columns
     stats_df.columns = stats_dict.keys()
     # select columns (only include stats that were actually computed)

